content/libraries/clients.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Client Libraries
+description: Client libraries handle communications between your software and the IOTA network.
+---
+
+# Client Libraries
+
+Client libraries handle communications between your software and the IOTA network. You can use a client library
+to [connect to a node](/iota.rs/examples/get_info)
+, [post a message](/iota.rs/examples/simple_message)
+, [send a signed transaction](/iota.rs/examples/transaction), and many other operations.
```

Comment:
I don't really like "...and many other operations" because my next question is "which operations exactly? Can I get the full list?"

> For example, you could use the client library to connect to a node, post a message, or send a signed transaction.

Reply:
```suggestion
Client libraries handle communications between your software and the IOTA network. For example, you can use a client library
to [connect to a node](/iota.rs/examples/get_info)
, [post a message](/iota.rs/examples/simple_message)
, [send a signed transaction](/iota.rs/examples/transaction).
```


the full list is in the client lib docs ;) , but it works either way.